src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +58,29 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
-
   return (
-    <>
-      <h4 className="admin-page-title">You are editing venue: {venue.name}</h4>
-      <BannerAdmin venueId={venueId} venue={venue} />
-      {isIframeVenue && <IframeAdmin venueId={venueId} venue={venue} />}
-    </>
+    <WithNavigationBar>
+      <div className="AdminPage">
+        <h4 className="AdminPage__title">
+          Current Announcement in Space Title
+        </h4>
+        {isShowBannerAdmin ? (
+          <BannerAdmin
+            venueId={venueId}
+            venue={venue}
+            onClose={hideBannerAdmin}
+          />
+        ) : (
+          <>
+            <AnnouncementMessage banner={venue?.banner} />
+            <AnnouncementOptions
+              banner={venue?.banner}
+              onEdit={showBannerAdmin}
+            />
+          </>
+        )}
```

Comment:
Is making this 1 or the other correct? Seems weird.. should there not be a 'neither' option?

Reply:
changed logic in 771b70a to return null in 1 case